frontend/styles/footer.css

```diff
@@ -0,0 +1,167 @@
+* 
+{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+footer
+{
+    border-top: 1.5px solid rgb(180, 180, 180);
+}
+
+.footer__logo-box
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 5rem;
+}
+
+.row
+{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    padding: 1.5rem 0;
+    margin: 1rem 5rem;
+}
+
+.col-1-of-2
+{
+    display: flex;
+    min-width: 500px;
+    max-width: 500px;
+    margin-right: 50px;
+}
+
+.col-2-of-2
+{
+    display: flex;
+    flex-direction: column;
+    margin-left: 50px;
+    min-width: 500px;
+    max-width: 500px;
+}
+
+.footer__navigation
+{
+    width: 100%;
+    padding: 18px 0 0 0;
+    border-top: 1px solid rgb(180, 180, 180);
+    font-family: 'Roboto', sans-serif;
+    font-weight: 800;
+    font-size: 17px;
+    color: rgb(70, 69, 69);
+}
+
+.footer__link:hover
+{
+    color: rgb(46, 94, 227);
+    font-size: 110%;
+}
+
+.footer__copyright
+{
+    width: 100%;
+    padding: 18px 0 0 0;
+    border-top: 1px solid rgb(180, 180, 180);
+    text-decoration: none;
+    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-weight: 500;
+    font-size: 17px;
+    color: rgb(70, 69, 69);
+    letter-spacing: 1px;
+    text-align: center;
+}
+
+.footer__list
+{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    min-width: max-content;
+    list-style-type: none;
+}
+
+.footer__item
+{
+    display: flex;
+    justify-content: space-between;
+    text-transform: uppercase;
+    text-decoration: none;
+}
+
+.footer__link
+{
+    text-decoration: none;
+    cursor: pointer;
+    transition: 350ms;
+    color: rgb(70, 69, 69);
+}
+
+.social
+{
+    display: flex;
+    justify-content: space-evenly;
+    text-align: center;
+    padding: 18px 40px 0 40px ;
+}
+
+.social>i
+{
+    font-size: 30px;
+    color: rgb(46, 94, 227);
+    transition: 350ms;
+}
+
+.social>i:hover
+{
+    cursor: pointer;
+    font-size: 40px;
+    color: rgb(27, 85, 244);
+}
+
+@media (max-width: 1135px) {
```

Comment:
Do write media queries for various screen sizes.
You can refer different screen sizes from base.css